### PR TITLE
[zh-cn] Update authorization page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 1]
```diff
@@ -190,23 +190,25 @@ Kubernetes 有时使用专门的动词以对额外的权限进行鉴权。例如
...
 
-* **Node** - 一个专用鉴权组件，根据调度到 kubelet 上运行的 Pod 为 kubelet 授予权限。
-  了解有关使用节点鉴权模式的更多信息，请参阅[节点鉴权](/zh/docs/reference/access-authn-authz/node/)。
-* **ABAC** - 基于属性的访问控制（ABAC）定义了一种访问控制范型，通过使用将属性组合
-  在一起的策略，将访问权限授予用户。策略可以使用任何类型的属性（用户属性、资源属性、
-  对象，环境属性等）。要了解有关使用 ABAC 模式的更多信息，请参阅
-  [ABAC 模式](/zh/docs/reference/access-authn-authz/abac/)。
-* **RBAC** - 基于角色的访问控制（RBAC）是一种基于企业内个人用户的角色来管理对
-  计算机或网络资源的访问的方法。在此上下文中，权限是单个用户执行特定任务的能力，
+* **Node** - 一个专用鉴权模式，根据调度到 kubelet 上运行的 Pod 为 kubelet 授予权限。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:193]:
```suggestion
* **Node** —— 一个专用鉴权模式，根据调度到 kubelet 上运行的 Pod 为 kubelet 授予权限。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 2]
```diff
@@ -190,23 +190,25 @@ Kubernetes 有时使用专门的动词以对额外的权限进行鉴权。例如
...
+  要了解有关使用节点鉴权模式的更多信息，请参阅[节点鉴权](/zh-cn/docs/reference/access-authn-authz/node/)。
+* **ABAC** - 基于属性的访问控制（ABAC）定义了一种访问控制范型，通过使用将属性组合在一起的策略，
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:195]:
```suggestion
* **ABAC** —— 基于属性的访问控制（ABAC）定义了一种访问控制范型，通过使用将属性组合在一起的策略，
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 3]
```diff
@@ -190,23 +190,25 @@ Kubernetes 有时使用专门的动词以对额外的权限进行鉴权。例如
...
+  将访问权限授予用户。策略可以使用任何类型的属性（用户属性、资源属性、对象，环境属性等）。
+  要了解有关使用 ABAC 模式的更多信息，请参阅
+  [ABAC 模式](/zh-cn/docs/reference/access-authn-authz/abac/)。
+* **RBAC** - 基于角色的访问控制（RBAC）
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:199]:
```suggestion
* **RBAC** —— 基于角色的访问控制（RBAC）
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 4]
```diff
@@ -190,23 +190,25 @@ Kubernetes 有时使用专门的动词以对额外的权限进行鉴权。例如
...
+  在此上下文中，权限是单个用户执行特定任务的能力，
   例如查看、创建或修改文件。要了解有关使用 RBAC 模式的更多信息，请参阅
-  [RBAC 模式](/zh/docs/reference/access-authn-authz/rbac/)。
-  * 被启用之后，RBAC（基于角色的访问控制）使用 `rbac.authorization.k8s.io` API 组来
-    驱动鉴权决策，从而允许管理员通过 Kubernetes API 动态配置权限策略。
+  [RBAC 模式](/zh-cn/docs/reference/access-authn-authz/rbac/)。
+  * 被启用之后，RBAC（基于角色的访问控制）使用 `rbac.authorization.k8s.io` API
+    组来驱动鉴权决策，从而允许管理员通过 Kubernetes API 动态配置权限策略。
   * 要启用 RBAC，请使用 `--authorization-mode = RBAC` 启动 API 服务器。
 * **Webhook** - WebHook 是一个 HTTP 回调：发生某些事情时调用的 HTTP POST；
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:207]:
```suggestion
* **Webhook** —— WebHook 是一个 HTTP 回调：发生某些事情时调用的 HTTP POST；
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 5]
```diff
@@ -94,38 +95,38 @@ Kubernetes reviews only the following API request attributes:
...
 For example, a `GET` request to endpoints like `/api` or `/healthz` would use `get` as the verb.
 -->
 ## 确定请求动词  {#determine-the-request-verb}
 
 **非资源请求**
 
-对于 `/api/v1/...` 或 `/apis/<group>/<version>/...` 之外的端点的请求被
-视为“非资源请求（Non-Resource Requests）”，并使用该请求的 HTTP 方法的
-小写形式作为其请求动词。
+对于 `/api/v1/...` 或 `/apis/<group>/<version>/...` 之外的端点的请求被视为
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:127]:
```suggestion
对于 `/api/v1/...` 或 `/apis/<group>/<version>/...`
之外的端点的请求被视为“非资源请求（Non-Resource Requests）”，
并使用该请求的 HTTP 方法的小写形式作为其请求动词。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md (hunk 2 above)
Comment (howieyuen) [reply content/zh-cn/docs/reference/access-authn-authz/authorization.md:196]: missing this one ?

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 6]
```diff
@@ -48,7 +48,15 @@ Kubernetes authorizes API requests using the API server. It evaluates all of the
 request attributes against all policies and allows or denies the request. All
 parts of an API request must be allowed by some policy in order to proceed. This
 means that permissions are denied by default.
+-->
+## 确定是允许还是拒绝请求
```
Comment (ydFu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:52]:
```suggestion
## 确定是允许还是拒绝请求  {#determine-whether-a-request-is-allowed-or-denied}
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**